meta-resin-common/recipes-support/resin-device-pinning/resin-device-pinning/resin-device-pinning

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+LOG_FILE=/var/log/resin-device-pinning-curl.log
+PINNING_CONF=/etc/resin-device-pinning.conf
+
+help () {
+    cat << EOF
+Pinning a resin device to a specific release commit
+resin-device-pinning [options]
+
+Options:
+    -h, --help
+        Display this help and exit.
+
+    -c, --config-path CONFIG_PATH
+        Use a non default config.json file.
+        This argument is passed to resin-vars script.
+EOF
+}
+
+# Parse arguments
+while [ $# -gt 0 ]; do
+    key=$1
+    case $key in
+        -h|--help)
+            help
+            exit 0
+            ;;
+        -c|--config-path)
+            RESIN_VARS_ARGS="$RESIN_VARS_ARGS --config-path $2"
+            shift
+            ;;
+        *)
+            echo "[WARNING] $0 : Argument '$1' unknown. Ignoring."
+            ;;
+    esac
+    shift
+done
+
+# shellcheck disable=SC2086
+source /usr/sbin/resin-vars $RESIN_VARS_ARGS
+
+if [ -f ${PINNING_CONF} ]; then
+  source /etc/resin-device-pinning.conf
```

Comment:
Who generates this file? I assume RELEASE_ID is expected to come from here. In any way, this script's description needs much more information.

Reply:
resin-preload generates this file

Reply:
In the rootfs?

Reply:
yes